### DISCUSSION TITLE
Finding loss statistic in FastSurvivalSVM

### DISCUSSION THREAD

**Discussion by kingjosephm** [sebp/scikit-survival#453] [General]
Great work! I'm looking for the loss at the end of training with the FastSurvivalSVM model and was hoping someone could point me to this object. In the documentation online I see there's mention of `model.optimizer_result_`, which in my case outputs something like:

message: Optimization terminated successfully.
 success: True
  status: 0
     fun: 541.9073216380415
       x: [ 5.031e+00 -9.981e-02 ... -8.155e-01 -1.562e+00]
     nit: 19
     jac: [-6.072e-05  2.419e-05 ...  5.993e-05  5.601e-05]
    nfev: 21
    njev: 21
    nhev: 937
 timings: None

Scipy's [documentation](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.OptimizeResult.html) suggests the value for the parameter `func` is the value for the objection function, is this what I'm looking for?

Thanks for your help!

**Comment by sebp** [comment]
That's correct, `fun` is the value of the loss function.
